drgn/helpers/linux/__init__.py

```diff
@@ -33,9 +33,9 @@
 import importlib
 import pkgutil
 
-__all__ = []
+__all__ = []  # type: ignore
 for _module_info in pkgutil.iter_modules(
-    __path__,  # type: ignore[name-defined]  # python/mypy#1422
+    __path__,  # python/mypy#1422
```

Comment:
It's super annoying that mypy warns for unused `type: ignore` comments which were necessary on older versions. Oh well.

The "comment on the comment" was the explanation for the `type: ignore`, so you can drop it now (and then all of the arguments can fit on one line):

```python
for _module_info in pkgutil.iter_modules(__path__, prefix=__name__ + "."):
```

Reply:
Oh, I guess we can add `--no-warn-unused-ignores` to the `mypy` command line. Feel free to do that instead of removing this comment, whatever you think is more useful.

Reply:
I'll add the argument so we maximize compatibility.